<ui version="4.0" >
 <class>Program</class>
 <widget class="QDialog" name="Program" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Program</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="programName" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Program</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="channel" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Start:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>End</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QDateEdit" name="beginDate" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="endDate" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QTimeEdit" name="beginHour" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="endHour" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QCheckBox" name="checkBoxAvant" >
              <property name="text" >
               <string>start</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAjouter" >
              <property name="text" >
               <string>end</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLineEdit" name="before" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="add" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="text" >
               <string>10</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>minutes before</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>minutes after</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Format filename:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filename" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>File created:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="overviewFilename" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%n&lt;/span> : Channel name  &lt;span style=" font-weight:600;">%t&lt;/span> : Program Title  &lt;span style=" font-weight:600;">%y&lt;/span> : Year &lt;span style=" font-weight:600;">%m&lt;/span> : Month  &lt;span style=" font-weight:600;">%d&lt;/span> : Day&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="desc" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="viewButton" >
       <property name="text" >
        <string>&amp;View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>&amp;Program</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="labelInfo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>Program</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>617</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel" >
     <x>342</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Program</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>480</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel" >
     <x>306</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
